src/App.jsx

```diff
@@ -1,11 +1,66 @@
 import { useState } from "react";
 import "./App.css";
-
+import {AiFillPlusCircle} from "react-icons/ai"
+import Todo from "./components/Todo";
+import { nanoid } from "nanoid";
 
 const App =()=> {
+  const [inputValue, setInputValue] = useState("");
+  const [todo, setTodo] = useState([])
+
+  const handleInput =(event) => {
+    setInputValue(prevInput=> (prevInput, event.target.value) )
+  }
+
+  const handleDelete =(id) =>{
+    setTodo(prevTodo => prevTodo.filter((obj)=> obj.props.id != id))
```

Comment:
You can use  !== instead of != to avoid type coercion issues. So, you can change the comparison operator from != to !==. thought what you did also is good.